src/main/java/InputData/songInputData.java

```diff
@@ -1,7 +1,58 @@
 package InputData;
 
+import Entities.ArtistUser;
+import Entities.Song;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.util.List;
+
 public class songInputData {
+    private final Song song;
+
+    public songInputData(Song song){
+        this.song = song;
+    }
+
+    public Song getSong() {
+        return song;
+    }
+
+    public int getId() {
```

Comment:
Thanks for implementing these functions! I think it's a great idea as opposed to calling the Song entity's .getID function itself.